.github/workflows/tests.yml

```diff
@@ -118,3 +118,12 @@ jobs:
           cluster-name: "${{ steps.commit_info.outputs.cluster_name }}"
           aws-region: ${{ env.TESTS_AWS_REGION }}
           s3-backend-bucket: ${{ env.TF_S3_BUCKET }}
+
+      - name: Notify in Slack in case of failure
+        id: slack-notification
+        if: failure() && (github.event_name == 'schedule' || github.event_name == 'workflow_dispatch')
```

Comment:
do we care about the `workflow_dispatch` even being reported?
May be someone trying something out and is usually triggered with intention.